materialize-mysql/driver.go

```diff
@@ -258,30 +273,28 @@ func (c client) AddColumnToTable(ctx context.Context, dryRun bool, tableIdentifi
 	var query string
 
 	err := c.withDB(func(db *stdsql.DB) error {
-		var conn, err = db.Conn(ctx)
-		if err != nil {
-			return err
-		}
-
-		rows, err := conn.QueryContext(ctx, "SELECT * FROM information_schema.columns WHERE table_schema=DATABASE() AND table_name=? AND column_name=?", tableIdentifier, columnIdentifier)
-		if err != nil {
-			return err
-		}
-
-		var exists = rows.Next()
-
-		if !exists {
-			query = fmt.Sprintf(
-				"ALTER TABLE %s ADD COLUMN %s %s;",
-				tableIdentifier,
-				columnIdentifier,
-				columnDDL,
-			)
-
-			if !dryRun {
-				if err := c.withDB(func(db *stdsql.DB) error { return sql.StdSQLExecStatements(ctx, db, []string{query}) }); err != nil {
-					return err
+		query = fmt.Sprintf(
+			"ALTER TABLE %s ADD COLUMN %s %s;",
+			tableIdentifier,
+			columnIdentifier,
+			columnDDL,
+		)
```

Comment:
The previous approach was failing when the identifier was quoted, because when using placeholders and parameters, identifiers should not be quoted. So instead I'm just trying out the query and if it fails with the specific error of "column exists", we just ignore that error.